README.md

```diff
@@ -2,14 +2,15 @@
 
 ## Inputs
 
-| Input              | Required | Default | Description                                                            |
-| ------------------ | :------: | :-----: | ---------------------------------------------------------------------- |
-| `GOOGLE_USER_MAIL` |  `true`  |         | The user ID to authenticate the service account user.                  |
-| `GOOGLE_USER_KEY`  |  `true`  |         | The users private key to authenticate the service account user.        |
-| `SPREADSHEET`      |  `true`  |         | The spreadsheet ID to append the data to.                              |
-| `DATA_ENV_NAME`    |  `true`  |         | The environment variable containing the data to upload in CSV format.  |
-| `TABLE_START_CELL` | `false`  |  `A1`   | The cell that defines the "table anchor" in the format `sheetName!A1`. |
-| `UPDATE`           | `false`  | `false` | Update or append data.                                                 |
+| Input              | Required | Default | Description                                                                                              |
+| ------------------ | :------: | :-----: | -------------------------------------------------------------------------------------------------------- |
+| `GOOGLE_USER_MAIL` | `true`  |          | The user ID to authenticate the service account user.                                                    |
+| `GOOGLE_USER_KEY`  | `true`  |          | The users private key to authenticate the service account user.                                          |
+| `SPREADSHEET`      | `true`  |          | The spreadsheet ID to append the data to.                                                                |
+| `DATA_FILE_PATH`   | `false` |          | The csv file containing the data to upload. Use either this or DATA_ENV_NAME.                            |
+| `DATA_ENV_NAME`    | `false` |          | The environment variable containing the data to upload in CSV format. Use either this or DATA_FILE_PATH. |
+| `TABLE_START_CELL` | `false` | A1       | The cell that defines the "table anchor" in the format `sheetName!A1`.                                   |
+| `UPDATE`           | `false` | false    | Update or append data.                                                                                   |
```

Comment:
It would be worth mentioning the precedence of `DATA_FILE_PATH` over `DATA_ENV_NAME`.
An explicit mention is always good for the end users. Thanks!

Example:

**NOTE**: If `DATA_FILE_PATH` is configured, `DATA_ENV_NAME` will be ignored.